README.md

```diff
@@ -9,3 +9,27 @@ Get a list of the PRs that got closed today, and email it to the POs (_this last
 ```
 PO_TOKEN=... bundle exec ruby todays-prs.rb
 ```
+
+
+## 'release-pdf.sh'
+
+# pre-steps
+//get ruby
+curl -L https://get.rvm.io | bash -s stable --auto-dotfiles --autolibs=enable --ruby
+rvm install "ruby-2.3.1"
+rvm use 2.3.1
+gem install bundler
+
+//get other dependencies
+*install node*
+sudo npm install mdpdf -g
+sudo npm install -g phantomjs-prebuilt --unsafe-perm=true  --allow-root
```

Comment:
Another Markdown note. You can wrap text that you'd like to be rendered as code either in three backticks (`) or by indenting it 4 spaces. The former is my preferred choice fwiw.